pkg/helmreconciler/reconciler.go

```diff
@@ -95,71 +100,53 @@ func (h *HelmReconciler) Reconcile() error {
 	// render charts
 	manifestMap, err := h.renderCharts(h.customizer.Input())
 	if err != nil {
-		err = errors.Wrap(err, fmt.Sprintf("error rendering charts"))
-		listenerErr := h.customizer.Listener().EndReconcile(h.instance, err)
```

Comment:
I am unclear if this lgoic needs to be retained to clean up the listeners in a failed reconcile..

Reply:
i don't think so - we're just reporting status to a listener. but it raises a valid point - if reconcile fails, what does that mean for component status? it should probably stay in "reconciling". have left a TODO for that.